src/main/java/hudson/plugins/git/UserMergeOptions.java

```diff
@@ -134,5 +156,15 @@ public ListBoxModel doFillMergeStrategyItems() {
                 m.add(strategy.toString(), strategy.toString());
             return m;
         }
+
+        public FormValidation doCheckBranchToMerge(@QueryParameter String mergeTarget, @QueryParameter String mergeSource) {
```

Comment:
for Stapler methods it's recommended to prohibit external usage via <code>@Restricted</code>, because the UI may easily change later